### DISCUSSION TITLE
评测qa形式自定义数据集，命令行启动，为什么固定使用SizePartitioner，导致infer的结果都被切分了，能不能不切分啊

### DISCUSSION THREAD

**Discussion by pzfok** [open-compass/opencompass#1119] [Q&A]
05/07 23:25:22 - OpenCompass - DEBUG - An `SizePartitioner` instance is built from registry, and its implementation can be found in opencompass.partitioners.size
05/07 23:25:22 - OpenCompass - DEBUG - Key eval.runner.task.judge_cfg not found in config, ignored.
05/07 23:25:22 - OpenCompass - DEBUG - Key eval.runner.task.dump_details not found in config, ignored.
05/07 23:25:22 - OpenCompass - DEBUG - Key eval.given_pred not found in config, ignored.
05/07 23:25:22 - OpenCompass - DEBUG - Additional config: {}
05/07 23:25:22 - OpenCompass - INFO - Partitioned into 10 tasks.
05/07 23:25:22 - OpenCompass - DEBUG - Task 0: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_0]
05/07 23:25:22 - OpenCompass - DEBUG - Task 1: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_1]
05/07 23:25:22 - OpenCompass - DEBUG - Task 2: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_2]
05/07 23:25:22 - OpenCompass - DEBUG - Task 3: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_3]
05/07 23:25:22 - OpenCompass - DEBUG - Task 4: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_4]
05/07 23:25:22 - OpenCompass - DEBUG - Task 5: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_5]
05/07 23:25:22 - OpenCompass - DEBUG - Task 6: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_6]
05/07 23:25:22 - OpenCompass - DEBUG - Task 7: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_7]
05/07 23:25:22 - OpenCompass - DEBUG - Task 8: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_8]
05/07 23:25:22 - OpenCompass - DEBUG - Task 9: [opencompass.models.huggingface.HuggingFace_test_internlm/test_qa_5000_9]

**Comment by bittersweet1999** [comment]
可以提供您的命令行脚本嘛，看起来您使用了--debug模式，如果要不切分可以考虑让size partitioner的size变大或者使用naive partitioner。
